addons/l10n_cl/models/res_partner.py

```diff
@@ -21,3 +22,31 @@ class ResPartner(models.Model):
              '2 - Fees Receipt Issuer (Applies to suppliers who issue fees receipt)\n'
              '3 - End consumer (only receipts)\n'
              '4 - Foreigner')
+
+    def _format_vat_cl(self, values):
+        identification_types = [self.env.ref('l10n_latam_base.it_vat').id, self.env.ref('l10n_cl.it_RUT').id,
+                                self.env.ref('l10n_cl.it_RUN').id]
+        partner_country_is_chile = values.get('country_id') == self.env.ref('base.cl')
```

Comment:
Should not self.env.ref('base.cl') be a record, while country_id is an id?  I actually would like to avoid the not values.get('country_id') because this will be applied anywhere.  I was more thinking about something like this: 

`partner_country_is_chile = (values.get('country_id') == self.env.ref('base.cl').id) or (values['latam_identification_type'] and self.env['latam.identification.type...'].browse(....).country_id == self.env.ref('base.cl'))`

and then avoid the no country case, because by default you will have RUT as identification type which will apply the validation.  Could you please still check this one still, so we can get this merged?  Sorry for the confusion with the values.get().

Reply:
@Danisan